pom.xml

```diff
@@ -47,12 +51,12 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-annotations</artifactId>
-      <version>2.7.0-SNAPSHOT</version>
+      <version>${jackson.version}</version>
```

Comment:
This is not correct, as we intentionally want to keep annotations version at .0 (ideally, I'd only release minor versions like "2.6", but due to dev preferences, patch versions are released despite no changes going in).